andela/bp-andela-media
# #161593686 Rename an  album

#### Note: Please only review the changes brought in by my commit.
#### What does this PR do?
Allows a user to rename an album 
#### Description of Task to be completed?
Enbale the user to change the name of an album created.
#### How should this be manually tested?
* Run the server `$ rails s`
* Open postman
* Select method `POST` and url `http://127.0.0.1:3000/album`
* create an album using the JSON object below
```
{
   "title": "Album title",
   "description": "This is a test album",
   "user_id": "cfe28054-0468-47b4-ac78-bf2b5c6838cf",
   "tag_id": "2"
}
```
* Copy the `album id` 
* Change url to `http://127.0.0.1:3000/album/<album_id>`
* Select `PUT` method
* Create a JSON object with any title in the body as shown below
```
{
   "title": "New title"
}
```
* Click send
* Ensure a response with message `Album title updated successfully` is returned

#### What are the relevant pivotal tracker stories?
[#161593686](https://www.pivotaltracker.com/story/show/161593686)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="568" alt="screen shot 2018-12-04 at 16 36 11" src="https://user-images.githubusercontent.com/26734746/49445345-c552af00-f7e2-11e8-8827-1980091b0812.png">
